lib/std/event/loop.zig

```diff
@@ -817,8 +817,8 @@ pub const Loop = struct {
                 @atomicStore(i32, &self.os_data.fs_queue_item, 1, AtomicOrder.SeqCst);
                 const rc = os.linux.futex_wake(&self.os_data.fs_queue_item, os.linux.FUTEX_WAKE, 1);
                 switch (os.linux.getErrno(rc)) {
-                    0 => {},
-                    os.EINVAL => unreachable,
+                    @intToEnum(os.linux.Errno, 0) => {},
```

Comment:
it seems clear to me that there should be a tag value of Errno which represents "no error" and has value 0.

Reply:
I thought that.... and then thought that `getErrno` returning an optional would work better: what do you think of that?

Reply:
I don't think that's the correct data layout. These ABIs have 0 as an *in-bound value*.

Reply:
Lacking that for now, #3806 could put `0` out of range, and hence make it so `0` is `null`